### PR TITLE
SS-01/Setup_django: setup Django and common project

### PR DESCRIPTION
**Pull request by Moryakprod**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Pull base image
+FROM python:3.10.4
+
+# Set environment variables
+ENV PIP_DISABLE_PIP_VERSION_CHECK 1
```
Comment (YevheniiaSmyrnova) [inline Dockerfile:5]: Why do you need this?

File: docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    build: .
+    command: python3 ./manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/code
+    ports:
+      - "8000:8000"
+    depends_on:
+      - postgres
+    environment:
+      - USER_NAME=${POSTGRES_USER}
```
Comment (YevheniiaSmyrnova) [inline docker-compose.yml]: Why you need this here?

**Comment by YevheniiaSmyrnova** [comment]
Need to remove pycache

File: service_station/settings.py [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+
+# Database
+# https://docs.djangoproject.com/en/4.0/ref/settings/#databases
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'service_station',
+        'USER': os.getenv('USER_NAME'),
```
Comment (YevheniiaSmyrnova) [inline service_station/settings.py]: For it you use USER_NAME or POSTGRES_USER?

Comment (YevheniiaSmyrnova) [inline service_station/settings.py]: Try to use environ instead ofos.getenv

File: Dockerfile (hunk 1 above)
Comment (Moryakprod) [reply Dockerfile:5]: I use this to turn off checking version of pip and to disable notifications about available of new version pip.

**Review by YevheniiaSmyrnova — APPROVED** [review]

**Review by YevheniiaSmyrnova — APPROVED** [review]

**Pull request merged**